index.html

```diff
@@ -9,20 +9,20 @@
 </head>

```

Comment:
You'll need to specify the viewport for browsers, else mobile browsers will scale down the site to fit in on the screen.

`<meta name="viewport" content="width=device-width, initial-scale=1">` should work

You can use chrome's mobile emulator to test with, though if you have a physical device you can test with that with something like https://www.npmjs.com/package/http-server